main.go

```diff
@@ -175,6 +175,33 @@ func doesDaemonSetExist(daemonSetName, namespace string) bool {
 	return err == nil
 }
 
+func DaemonSetReady(daemonSetName, namespace, image string) bool {
+	const hoursPerWeek = 168
```

Comment:
Why this number?

Reply:
A better name would be IsDaemonSetReady

Reply:
No, that is fine. Just add a comment next to this number (// 7 days), because the number was not obvious. May be the name of the variable should be hoursInWeek.